.github/workflows/turbopack-nightly-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Turobpack Nightly Release
+
+on:
+  schedule:
+    - cron: "15 8 * * *" # Run every day at 00:15 PST
+  workflow_dispatch:
+
+jobs:
+  turbopack_nightly:
+    name: Cut nightly Turbopack release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout latest commit that passes CI
+        uses: talentpair/last-green-commit-action@d95cfa8
+        id: green_commit
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Tag nightly
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          commit_sha: ${{ steps.green_commit.outputs.result }}
+          tag_prefix: "turbopack-"
+          # Override the default conventional commit messages
+          # so that we always generate patch releases.
+          # This would be easier if we could filter Turbopack
+          # commits from Turborepo, but that'd require a custom
+          # action.
+          custom_release_rules: ""
+          # Ensure we generate prerelease versions, not stables.
+          release_branches: ""
+          pre_release_branches: "main"
+          append_to_pre_release_tag: "alpha"
+
+      - name: Create GitHub release
+        uses: ncipollo/release-action@v1
+        with:
+          tag: ${{ steps.tag_version.outputs.new_tag }}
+          name: Turbopack Release ${{ steps.tag_version.outputs.new_tag }}
+          body: ${{ steps.tag_version.outputs.changelog }}
```

Comment:
We probably don't want to include turborepo changes here...

Reply:
That's the difficult part, because we don't currently have a good list of things that are Turbopack vs Turborepo. Splitting this into better defined directories would be helpful.

Reply:
Yeah. I guess we probably don't need a changelog for the nightlies anyway

Reply:
Removed